### DISCUSSION TITLE
Could `hash` type be `Record<string, any>`

### DISCUSSION THREAD

**Discussion by dargmuesli** [handlebars-lang/handlebars.js#2000] [Ideas]
Could this `any` be turned to `Record<string, any>`?

https://github.com/handlebars-lang/handlebars.js/blob/ae83edd5c490ccaa41e3d345e5068378e6207837/types/index.d.ts#L48

`Record<string, string>` would be too strict, I guess.

**Comment by jaylinski** [comment]
See #2001.
